x-pack/plugins/security_solution/cypress/integration/detection_rules/indicator_match_rule.spec.ts

```diff
@@ -108,7 +108,7 @@ import { loginAndWaitForPageWithoutDateRange } from '../../tasks/login';
 import { goBackToAllRulesTable } from '../../tasks/rule_details';
 
 import { ALERTS_URL, RULE_CREATION } from '../../urls/navigation';
-import { DEFAULT_THREAT_MATCH_QUERY } from '../../../common/constants';
+const DEFAULT_THREAT_MATCH_QUERY = '@timestamp >= "now-30d"';
```

Comment:
@MadameSheema for some reason cypress is mad about this import. i kept getting that error about `memoizeOne`: https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/156245/execution/node/532/log/?consoleFull